source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Core/DateTime/InstantExtensionsTests.cs

```diff
@@ -23,7 +23,7 @@ namespace GreenEnergyHub.Charges.Tests.Core.DateTime
     public class InstantExtensionsTests
     {
         [Fact]
-        public void TimeOrEndDefault_WhenInstantIsNotNull_ReturnsInput()
+        public void TimeOrEndDefault_WhenInstantIsNotNull_InReturnsInput()
```

Comment:
Should it have been `_ItReturnsInput` ?
```suggestion
        public void TimeOrEndDefault_WhenInstantIsNotNull_ItReturnsInput()
```